src/mongoClusters/commands/exportDocuments.ts

```diff
@@ -48,14 +50,20 @@ export async function mongoClustersExportQueryResults(
     let documentCount = 0;
 
     // Wrap the export process inside a progress reporting function
-    await runExportWithProgressAndDescription(node.id, async (progress, cancellationToken) => {
-        documentCount = await exportDocumentsToFile(
-            docStream,
-            filePath,
-            progress,
-            cancellationToken,
-            docStreamAbortController,
-        );
+    await callWithTelemetryAndErrorHandling('cosmosDB.mongoClusters.exportDocuments', async (actionContext) => {
+        await runExportWithProgressAndDescription(node.id, async (progress, cancellationToken) => {
+            documentCount = await exportDocumentsToFile(
+                docStream,
+                filePath,
+                progress,
+                cancellationToken,
+                docStreamAbortController,
+            );
+        });
+
+        actionContext.telemetry.properties.source = props?.source;
```

Comment:
In fact this probably doesn't matter, when running KQL queries this will then also show as empty no matter what gets stored. Hence if we expect undefined values here, we should account for it when we query it.